### PR TITLE
Adding SlicerIGSIO extension

### PR DESCRIPTION
**Pull request by Sunderlandkyl**
Relies on https://github.com/Slicer/Slicer/pull/1028
@lassoan

### CODE REVIEW

**Comment by pieper** [comment]
I have no idea what IGISO is (and a quick google doesn't seem to help).  Could you include a definition or link in the readme?

**Comment by Sunderlandkyl** [comment]
Right. Added a link to the IGSIO repository and gave an example of some functionality (video IO).

**Comment by pieper** [comment]
It's better, but I'd say you should have an explicit link this site, because it's the one that actually explains what is going on: http://igsio.github.io/

**Comment by lassoan** [comment]
This is our standard extension checklist:
- [ ] Extension has a reasonable name (not too general, not too narrow, suggests what the extension is for)
- [ ] Repository name is Slicer+ExtensionName
- [ ] Extension description summarizes in 1-2 sentences what the extension is usable (should be understandable for non-experts)
- [ ] Extension URL and revision (scmurl, scmrevision) is correct, consider using a branch name (master, release, …) instead of a specific git has to avoid re-submitting pull request whenever the extension is updated
- [ ] Extension icon URL is correct
- [ ] Screenshot URLs (screenshoturls) are correct, contains at least one
- [ ] Homepage URL points to valid webpage containing the followings:
  - [ ] Extension name
  - [ ] Short description: 1-2 sentences, which summarizes what the extension is usable for
  - [ ] At least one nice, informative image, that illustrates what the extension can do. It may be a screenshot.
  - [ ] Description of contained modules: at one sentence for each module
  - [ ] Tutorial: step-by-step description of at least the most typical use case, include a few screenshots, provide download links to sample input data set

This is a utility extension (not to be installed by end users, only to be installed by other extensions), so tutorials, etc. can refer to relevant end-user extensions.

**Comment by Sunderlandkyl** [comment]
@lassoan Ok, I think it should be more verbose and descriptive now.

File: SlicerIGSIO.s4ext [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+# url to icon (png, size 128x128 pixels)
+iconurl https://raw.githubusercontent.com/IGSIO/SlicerIGSIO/master/SlicerIGSIO.png
+
+# Give people an idea what to expect from this code
+#  - Is it just a test or something you stand behind?
+status 
+
+# One line stating what the module does
+description This is a utility extension that is developed as part of is developed as part of the Image Guided Surgery InterOperability (IGSIO) organization (see http://igsio.github.io/), and should be used as a dependency for other extensions (Ex. SlicerIGT) to provide access to tools and algorithms implemented by IGSIO (https://github.com/IGSIO/IGSIO).
```
Comment (lassoan) [inline SlicerIGSIO.s4ext]:
```suggestion
description This is a utility extension that is developed as part of the Image Guided Surgery InterOperability (IGSIO) organization (see http://igsio.github.io/), and should be used as a dependency for other extensions (Ex. SlicerIGT) to provide access to tools and algorithms implemented by IGSIO (https://github.com/IGSIO/IGSIO).
```

**Comment by lassoan** [comment]
Thank you, it looks good to me now - except the small type in the extension description. Please also update CMakeLists.txt file to be in sync with the s4ext file.

**Comment by lassoan** [comment]
Thank you! I'll merge this now.

**Pull request merged**